lib/src/base-styles/_nx-forms.scss

```diff
@@ -19,13 +19,18 @@
 
 .nx-form-row {
   @include container-horizontal;
+  align-items: flex-end;
+  display: flex;
   margin-bottom: $nx-spacing-xs;
 
   .nx-form-group {
-    display: inline-block;
     margin-right: $nx-spacing-l;
     margin-bottom: 0;
   }
+
+  .nx-btn-bar {
+    margin-bottom: $nx-spacing-l;
```

Comment:
`.nx-btn-bar` comes with a top margin which you'll probably want to clear out here.  It doesn't make a difference in your examples, but I imagine it would if the adjacent text input didn't have any label text.

Reply:
> `.nx-btn-bar` comes with a top margin which you'll probably want to clear out here. It doesn't make a difference in your examples, but I imagine it would if the adjacent text input didn't have any label text.

https://github.com/sonatype/sonatype-react-shared-components/pull/164/commits/a0b67d2c8c575c77fab8731458c7017943d4e0ec